index.js

```diff
@@ -10,4 +14,146 @@ navLinks.forEach((link) => {
   link.addEventListener('click', () => {
     menuNav.classList.toggle('menu-hidden');
   });
-});
\ No newline at end of file
+});
+
+const RECENT_WORKS = [
+  {
+    title: 'Multi-Post Stories Gain+Glory',
+    name: 'Keeping track of hundreds of components',
```

Comment:
**OPTIONAL**
- [ ] Please you can remove the object "name"  and keep "title" because "name" does not appear in the popup. Also to keep your work clean, DRY. Nice job 👍

Reply:
Yes Sir. Thank you very much